src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -67,12 +73,47 @@ class ActivityFeed extends React.Component<Props, State> {
         if (hasLoaded || hasMoreItems || hasNewItems || hasInputOpened) {
             this.resetFeedScroll();
         }
+
+        if (hasLoaded) {
+            this.logRenderMetrics();
+        }
+    }
+
+    /**
+     * Checks to see whether or not the Actvity Feed is still loading
+     *
+     * @returns {boolean} True if still loading
+     */
+    isLoading(): boolean {
+        const { feedItems, currentUser } = this.props;
+        return !feedItems || !currentUser;
+    }
+
+    /**
+     * Log time to meaningful render
+     *
+     * @param {User} currentUser - CurrentUser for the session
+     * @param {FeedItems} feedItems - Feed items populating the activity feed
+     */
+    logRenderMetrics(): void {
+        const { logger } = this.props;
+        const hasLoggedRender = logger.hasLoggedEvent(ACTIVITY_FEED_TAGS.ComponentName, EVENT_TIME_TO_RENDER);
```

Comment:
Can't we use `logUniqueMetric` which will do this check?

Reply:
Oh, it's to avoid the redundant `performance.mark()`. I'm curious if we should instead wrap performance.mark in a thin wrapper to avoid redundant marking, since each `mark()` call with the same tag will create ANOTHER mark